src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/ISymWrapperCore.cs

```diff
@@ -28,7 +28,7 @@ namespace System.Reflection.Emit
     // SymWrapperCore is never instantiated and is used as an encapsulation class.
     // It is our "ISymWrapper.dll" assembly within an assembly.
     //------------------------------------------------------------------------------
-    internal class SymWrapperCore
+    internal sealed class SymWrapperCore
```

Comment:
Should SymWrapperCore be made a static class?

Reply:
Seems like it probably can be.

Reply:
I think there is CA1052 rule for that ;-)

Reply:
I postponed turning that on in the past due to the number of changes required, but I'll take another look at it.